lib/plugins/staticFiles.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+var assert = require('assert-plus');
```

Comment:
If this plugin is for new versions only, maybe we can use let/const here?

Reply:
I did not want to tackle the eslint rule that disables usage of es6 constructs in code as part of this PR. (Refer: https://github.com/restify/node-restify/blob/master/.eslintrc.js#L12). Let's keep that for a future PR where we might convert vars -> consts/let in the codebase.

Reply:
👍